File: plugins/my_native_plugin/lib/models/spam_number.dart
class SpamNumber{
  String number;
  String description;
  int id;
  SpamNumber({
    required this.number,
    required this.description,
    required this.id,
  });

  Map<String, dynamic> toMap(){
    return {
      "number":number,
      "description":description,
      "id":id
    };
  }
}

File: plugins/my_native_plugin/android/src/main/kotlin/com/stopscam/antispam_plugin/data/local/entity/SpamNumber.kt
package com.stopscam.antispam_plugin.data.local.entity

import androidx.room.Entity
import androidx.room.PrimaryKey
import androidx.annotation.Keep

@Keep
@Entity
data class SpamNumber(
    @PrimaryKey val number: String,
    val description: String,
    val serverId: Int,
)